cas-server-integration-ehcache/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java

```diff
@@ -50,7 +51,7 @@
  * @author Andrew Tillinghast
  * @since 3.5
  */
-public final class EhCacheTicketRegistry extends AbstractDistributedTicketRegistry implements InitializingBean {
+public final class EhCacheTicketRegistry extends AbstractCrypticTicketRegistry implements InitializingBean {

```

Comment:
You choose the inheritance way, I expected some composition in which you would have a new `CrypticTicketRegistry` in front of a regular registry: `new CrypticTicketRegistry(ticketRegistry);` This way, you would not have changed the existing ticket registries, wouldn't you? Easier and more backward compatible.